### PR TITLE
docs(lab4): update job & add cronjob

### PR DESCRIPTION
**Pull request by bbbmj**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #https://github.com/caicloud/kube-ladder/issues/13

**Special notes for your reviewer**:

/cc @ddysher 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tutorials/resources/job.yaml [hunk 1]
```diff
@@ -10,6 +10,6 @@ spec:
     spec:
       containers:
       - name: pi
-        image: perl
+        image: cargo.caicloud.io/caicloud/perl:latest
```
[the remark this answers is not in the document]
Comment (bbbmj) [reply tutorials/resources/job.yaml]: why

[the remark this answers is not in the document]
Comment (ddysher) [reply tutorials/resources/job.yaml]: we dont provide any SLA for this image service.

[the remark this answers is not in the document]
Comment (bbbmj) [reply tutorials/resources/job.yaml]: ...

**Pull request merged**